docs/content/integrations/dagstermill/using-notebooks-with-dagster.mdx

```diff
@@ -146,47 +146,44 @@ When materialized, the `iris_kmeans_jupyter` asset will execute the notebook (`/
 
 ---
 
-## Step 3: Add a Dagster Repository and supply an I/O manager
+## Step 3: Add a Dagster Definitions object and supply an I/O manager
 
-We want to execute our Dagster asset and save the resulting notebook to a persistent location. This is called materializing the asset and to do this, we need to add the asset to a Dagster [repository](/concepts/repositories-workspaces/repositories).
+We want to execute our Dagster asset and save the resulting notebook to a persistent location. This is called materializing the asset and to do this, we need to add the asset to a Dagster [Definitions](/concepts/code-locations).
 
 Additionally, we need to provide a [resource](/concepts/resources) to the notebook to tell Dagster how to store the resulting `.ipynb` file. We'll use an [I/O manager](/concepts/io-management/io-managers) to accomplish this.
 
-Open the `/tutorial_template/repository.py` file and add the following code:
+Open the `/tutorial_template/__init__.py` file and add the following code:
 
 ```python
-# tutorial_template/repository.py
+# tutorial_template/__init__.py
 
+from dagster import load_assets_from_package_module, Definitions
 from dagstermill import ConfigurableLocalOutputNotebookIOManager
-from dagster import repository, with_resources
+
 from . import assets
 
-@repository
-def template_tutorial():
-    return
-        with_resources(
-            load_assets_from_package_module(assets),
-            resource_defs={
-                "output_notebook_io_manager": ConfigurableLocalOutputNotebookIOManager(),
-            },
-        ),
+defs = Definitions(
+    assets=load_assets_from_package_module(assets),
+    resources={
+        "output_notebook_io_manager": ConfigurableLocalOutputNotebookIOManager()
+    }
+)
+
 ```
 
 Let's take a look at what's happening here:
 
-- Using <PyObject object="with_resources"/>, we've provided resources to the assets in the repository. In this example, that's the <PyObject module="dagstermill" object="ConfigurableLocalOutputNotebookIOManager" /> resource.
+- Using <PyObject object="load_assets_from_package_module" />, we've imported all assets in the `assets` module. This approach allows any new assets we create to be automatically added to the Definitions instead of needing to manually add them one by one.
```

Comment:
```suggestion
- Using <PyObject object="load_assets_from_package_module" />, we've imported all assets in the `assets` module. This approach allows any new assets we create to be automatically added to the `Definitions` object instead of needing to manually add them one by one.
```